src/lib.rs

```diff
@@ -1,18 +1,35 @@
 #![doc = include_str!("../README.md")]
 
-mod bindings {
-    #![allow(non_upper_case_globals, non_snake_case, non_camel_case_types)]
-    #![allow(dead_code)]
-
-    include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
-}
-use bindings::*;
-
 use std::borrow::Cow;
 use std::sync::{Arc, Once};
 
 static INITIALIZED: Once = Once::new();
 
+#[cfg(all(target_os = "windows", target_arch = "x86_64"))]
+mod bindings_x86_64_pc_windows_msvc;
+#[cfg(all(target_os = "windows", target_arch = "x86_64"))]
+use bindings_x86_64_pc_windows_msvc::*;
```

Comment:
Nit: you might be able to have a single `use bindings::*` without duplicated `cfg` if you write `mod bindings_target_triple as bindings;`?

Reply:
Don't think that is possible, right?

Reply:
Right yes I was mistaken with `use` instead of `mod`. Probably the only alternative is the original `include!()` `cfg`'d, or `#[cfg_attr(target..., path = "bindings_target")] #[...] mod bindings;`.